cli/tests/unit/files_test.ts

```diff
@@ -118,7 +56,7 @@ Deno.test(async function readerIter() {
   const reader = new TestReader("hello world!");
 
   let totalSize = 0;
-  for await (const buf of Deno.iter(reader)) {
+  for await (const buf of iterateReader(reader)) {
```

Comment:
It seems like this test targets `Deno.Reader` rather than `Deno.iter()`. Hence, keeping the test. It's possible that `iterateReader()` will live on in some form, depending on how https://github.com/denoland/deno_std/issues/4120 is handled.

It's also possible that this test will become the responsibility of the Standard Library if the Standard Library takes the `Reader` interface under its wings.